BranchBrowser.py

```diff
@@ -755,24 +755,39 @@ def move_to_left(self):
 
 
     def update_repo_branches_right_listbox(self, event = None):
-        self.repo_branch_right_lb_info_map.clear()
-
-        # Get the selected value of the repo name combobox
+        # Get the selected values of the repo name and branch type comboboxes
         repo_name = self.repos_combobox.get()
         branch_type = self.branch_type_combobox.get()
+
+        if event == None or event.widget == self.repos_combobox:

```

Comment:
if not event, da budemo konzistentni

Reply:
Ispravi ovo

Reply:
Ovo je bilo pre, sada je drugacije. Sada izgleda ovako:
if event and event.widget == self.repos_combobox

Trebalo bi da je u redu, da ne treba da se stavlja not, jer se ne proverava da li je event == None